src/netbase.cpp

```diff
@@ -676,40 +676,36 @@ bool ConnectThroughProxy(const proxyType& proxy, const std::string& strDest, uin
     return true;
 }
 
-bool LookupSubNet(const std::string& strSubnet, CSubNet& ret, DNSLookupFn dns_lookup_function)
+bool LookupSubNet(const std::string& subnet_str, CSubNet& subnet_out)
```

Comment:
Thinking of it, a better API would be to return `option<CSubNet>` instead of a bool and output parameter. But not necessarily in this PR. It's more consistent with the other functions in `netbase.h` like this.

Reply:
Good idea. I have a branch of netbase follow-ups, could do it there.